### DISCUSSION TITLE
Is the basePath feature closer to becoming a reality?

### DISCUSSION THREAD

**Discussion by jamesmosier** [vercel/next.js#10438] [Help]
I know it's been asked a million times before, so if the answer is the same I understand, but looking at the issue #4998 (basePath support) I noticed a recent PR land with experimental support for basePaths (#9872) and then a follow up PR (#9988) which fixed a couple bugs. I know in the original PR it was mentioned that this feature very well could never see the light of day outside of behind the experimental flag. So my question is, is that still the case? If so, any particular reason it might not stick (i.e. support nightmare? complicates routing? something else?)? I'm always interested in hearing some of the behind the scenes discussions of the roadmap and also would love to see this feature land 😄

**Comment by timneutkens** [comment, marked as answer]
Just haven't had time to revisit yet. If you really need it we can work on it under a support contract to prioritize.

**Comment by jbsmith731** [comment]
@timneutkens can you explain what a support contract is?

**Comment by timneutkens** [comment]
We work with some companies to provide them support in adopting Next.js / build out certain features (eg zones is an example of this).

Feel free to reach out to enterprise@zeit.co
